### ISSUE TITLE
Bootstrap 4 support

### ISSUE THREAD

**Issue by alejandroiglesias** [designmodo/Flat-UI-Pro-Support#527]
Since Bootstrap 4 alpha was released I was wondering what your plans are to support it, specially since the new version is based on Sass.
Thanks

**Comment by ottz0** [comment]
I'm wondering the same

**Comment by JohnCampionJr** [comment]
Agreed; project hasn't been updated in over a year at this point.  Any news?

**Comment by lggdance** [comment]
Same thing here.. Any news??

**Comment by Aridio** [comment]
do they abandoned us? 
:(

**Comment by jaredtmartin** [comment]
I'm wondering the same thing.

**Comment by sandstones** [comment]
The silence is deafening...

**Comment by alejandroiglesias** [comment]
I've found recently a couple projects:

**Bootflat**
https://bootflat.github.io/
This one looks very very similar to Flat UI, is built with Bootstrap 3, and it's fully open source, so it can receive a contribution to update it.

**Flatkit**
http://flatfull.com/themes/flatkit/
This one is paid and doesn't look the same as Flat UI, but still a good alternative and supports Bootstrap 4 right away.

I hope this information is useful for you guys. I'm evaluating how much effort it would take to update Bootflat to Bootstrap 4. Will post here if there are any updates.

**Comment by ideosis** [comment]
Seems there is no one supporting flat-ui-pro v1.4.0? Datatables has some issues and even the colour of dropdown menu in inverse. Love the look and feel but customer support is missing.
